src/ast/dml.rs

```diff
@@ -487,6 +487,90 @@ pub struct Insert {
     pub insert_alias: Option<InsertAliases>,
 }
 
+impl Display for Insert {
```

Comment:
I'm thinking we want to reuse the display implementation in the corresponding insert/delete statement display as well? Otherwise the new code is largely a duplicate of what we do there, and we'd need identical test cases which might not be worth it

Reply:
Oh wow I had no idea these existed 😅 

Asking to learn: How would I reuse what's already here in the other statements? Or are you just saying that instead of this implementation, it'd be better for me to use construct a `Statement::Insert` if I needed to do some programmatic construction

Reply:
Ah so I was rather thinking, taking insert as an example, that if we have the display implementation here, then we should be able to replace [this code](https://github.com/sqlparser-rs/sqlparser-rs/blob/main/src/ast/mod.rs#L3402-L3484) with something roughly like
```rust
Statement::Insert(insert) => write!(f, "{insert}")
```
since conceptually the representations should be the same?